runtime/runtime.go

```diff
@@ -195,7 +195,9 @@ func Bootstrap() {
 		runtime.LockOSThread()
 		factory, _ := libcontainer.New("")
 		if err := factory.StartInitialization(); err != nil {
-			panic(err)
+			panic("error bootstraping container " +
```

Comment:
It probably should assert error to "permission denied" first. It might be some other error not related to SELinux.

Reply:
Done!